omnipaxos/src/storage.rs

```diff
@@ -28,13 +28,18 @@ pub trait Entry: Clone + Debug {
 #[allow(missing_docs)]
 pub struct StopSignEntry {
     pub stopsign: StopSign,
-    pub decided: bool,
+    pub idx: u64,
```

Comment:
just to be ultra clear:
```suggestion
    /// The log index of this StopSign
    pub log_idx: u64,
```